src/QMCWaveFunctions/Jastrow/codegen/user_jastrow.py

```diff
@@ -0,0 +1,515 @@
+from __future__ import print_function
+
+from sympy import *
```

Comment:
For sympy the standard seems to be to explicitly include every imported symbol rather than define a namespace abbreviation.   Done here - there aren't that many needed.